Formula/clingo.rb

```diff
@@ -26,10 +27,12 @@ class Clingo < Formula
   link_overwrite "bin/reify"
 
   def install
+    py3 = `which python3`.chomp
```

Comment:
There's only one possible answer for this so shelling out is a bit redundant.

Reply:
Are you suggesting to inline it like this? 

`"-DPYTHON_EXECUTABLE=#{which python3}",`

Reply:
No, I meant that we already know what the answer to `which python3` is because we only support using the one `python3` now. There's no way that `which python3` doesn't return `Formula["python"].opt_bin/"python3"`.

Reply:
Updated